index.php

```diff
@@ -1,14 +1,23 @@
 <?php get_header(); ?>
+	
+<!-- adding and sizing - posts with thumbnails -->
+
+	<?php
+	  add_action('init', 'setupshop_theme_init');
+		function setupshop_theme_init() {
+	  add_theme_support( 'post-thumbnails' ); 
+	  set_post_thumbnail_size( 100, 50, true );
+	} 
```

Comment:
This goes into your functions.php. Never put functions like these in your template files.